hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/records/Records.java

```diff
@@ -34,8 +34,8 @@ public interface Records {
 
     Id nextKey();
 
-    PathSet findPath(Id target, Function<Id, Boolean> filter,
-                            boolean all, boolean ring);
+    PathSet findPath(Id target, Function<Id, Boolean> filter, boolean all,
```

Comment:
prefer to wrap line before "boolean all"

Reply:
this line only 75 char. so i think wrap line before "boolean all" is better.